### PR TITLE
Fix spelling in spellbook usage message

### PR DESCRIPTION
**Pull request by coiax**
IT'S TIME FOR LINE LENGTHS TO MATTER AGAIN.

Specifically, we're changing the 'you' to a 'You'

### CODE REVIEW

**Comment by phil235** [comment]
your line length are ridiculously short, it makes reading the code harder.

**Comment by Jordie0608** [comment]
I never thought we'd be back to damn punchcarding lines again.

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -719,8 +719,12 @@
 		recoil(user)
 	else
 		user.mind.AddSpell(S)
-		user <<"<span class='notice'>you rapidly read through the arcane book. Suddenly you realize you understand [spellname]!</span>"
-		user.attack_log += text("\[[time_stamp()]\] <font color='orange'>[user.real_name] ([user.ckey]) learned the spell [spellname] ([S]).</font>")
+		user << "<span class='notice'>You rapidly read through \
+			the arcane book. Suddenly you realize you understand \
+			[spellname]!</span>"
+		user.attack_log += text("\[[time_stamp()]\] \
+			<font color='orange'>[user.real_name] ([user.ckey]) \
```
Comment (Jordie0608) [inline code/game/gamemodes/wizard/spellbook.dm]: `key_name()`; User attack logs don't really need to have the owner's name in them so you could just replace this with `has`.
<sub><sub><sub><sub>I wonder why orange</sub></sub></sub></sub>

**Comment by Bawhoppen** [comment]
when did you get back coiax

**Comment by optimumtact** [comment]
can you address jordies comment please?

**Comment by optimumtact** [comment]
ping me to reopen or make a new pr when you're available again

**Pull request closed without merging**